src/compose/images.ts

```diff
@@ -172,7 +172,7 @@ export function imageFromService(service: ServiceInfo): Image {
 export async function triggerFetch(
 	image: Image,
 	opts: FetchOptions,
-	onFinish = _.noop,
```

Comment:
A value is always passed in so I removed the default for clarity and improved the typing for specifically what we expect (as opposed to inheriting `_.noop`s typing which is pretty generic with `any`s)